### ISSUE TITLE
Template: Standard issue

### ISSUE THREAD

**Issue by xsaschako** [DLR-SC/memilio#534]
We need a really simple template for standard issues. That way we don't need to pick the tiny link to open blank issue, which is the most common one for us working on the project.

Suggested names?
Standard issue.
Simple Issue.

**Comment by xsaschako** [comment]
@mknaranja Do you have a nice name suggestion?

**Comment by mknaranja** [comment]
I am not in favor of this. Everyone who is familiar with our repository can directly click the link below and he or she knows how detailed the description needs to be. We should not offer to every new user write short non-meaningful issues.

**Comment by xsaschako** [comment]
I agree and i think i was a little to eager to open this issue as a standart issue is certainly not the best way to handle this. I looked at our issues and the most common one seems to be something like “Enhancement or Feauture Issue” which is partly covered by the feature request template. But at least for me, I don't use that because it doesn't quite hit the nail. I also looked through several highly active GitHub Repos and found some interesting templates, which maybe we could use, e.g., especially [ChatGPT](https://github.com/lencx/ChatGPT/issues/new/choose). Many other repositories don't use such a development issue because they don't follow the issue <-> pull request format we do. I would find such an template helpful as it could remind me for example to be more specific, which i certainly not always are. I proposed four templates, which we may incorporate, ofcourse the emojis can be discussed ;-).
